test/test_suit/test_schema_mapping.py

```diff
@@ -88,9 +88,14 @@ def verify(self, round):
             if field == "RECORD_METADATA":
                 continue;
             if type(res[res_col[field]]) == str:
+                print(res[res_col[field]])
+                print(self.gold[field])
                 # removing the formating created by sf
                 assert ''.join(res[res_col[field]].split()) == self.gold[field]
             else:
+                print(type(res[res_col[field]]))
+                print(res[res_col[field]])
+                print(self.gold[field])
```

Comment:
nit, we can just print the value if the assert fails?

Reply:
yes I will remove this..